vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md

```diff
@@ -1,136 +1,101 @@
+6. Enter the `Code` and `Name` fields as required. 
+7. Complete the Model HTML layout in Velocity. The Inline Editing Assist tool is available by using the Ctl-Space buttons on your keyboard. Your image tag should look something like this: 
+`<img src="$content.your-attribute-code.getImagePath("0")"/> `
+8. Click `Save`. The new Content Model is complete and should appear in the list to be used anywhere in your application.  
 
-- Create a content type with an image attribute.
-![image](./img/cms_content_type_image.png)
-- Create a content based on the created  type.
-![image](./img/cms_content_with_image.png)
-- Under the *Content Attributes* section at the bottom of the page, click the *Add* button beside the image attribute. It will then open a modal that allows you to select an asset by clicking the *Use* action.
-![image](./img/cms_content_image_add.png)
 
-## Configuring Allowed File Extensions for Upload
+## Configure File Extensions for Upload
 
-The types of files that can be uploaded to an Entando application are controlled by
-configuration in the server side of the app builder. If you're using the quickstart this will be the
-`entando-de-app`.
+The types of files that can be uploaded to an Entando Application are controlled by the
+configuration of the server side of the App Builder. If you're using the quickstart version, this is the`entando-de-app`.
 
-There are two properties that control these settings and they are set in `src/main/conf/systemParams.properties` and are a comma delimitted list of values. The property keys are
+There are two properties that control these settings and they are set in `src/main/conf/systemParams.properties` and are a comma delimitted list of values. The property keys are:
 
-- `jacms.imageResource.allowedExtensions=`
-- `jacms.attachResource.allowedExtensions=`
+* `jacms.imageResource.allowedExtensions=`
+* `jacms.attachResource.allowedExtensions=`
 
-If you are configuring the upload for both the App Builder and the legacy Admin Console you should include
-both the MIME type of the asset and the file extension. The APIs that support resources will check MIME type of the uploaded asset.
+To configure the uploads for the App Builder and Admin Console, include both the MIME type and file extension of the asset. The APIs that support resources will check the MIME type of the uploaded asset.
 
-An example configuration for images.
+An example configuration for images:
 - `jacms.imageResource.allowedExtensions=jpg,jpeg,png,svg,svg+xml`
 
 Where `svg+xml` is the MIME type for an svg image.
 
-The `systemParams.properties` file is bundled into the WAR and the image for the application so you will need to create and deploy image to reflect changes for these properties in your application.
+The `systemParams.properties` file is bundled into the WAR and the image for the application, so you will need to create and deploy an image to reflect changes for these properties in your application.
```

Comment:
what is a WAR? should spell out acronym?

"you need to" -> no "will" (present tense)

"to" these properties, not "for"